src/pages/manage/productions/create/index.page.js

```diff
@@ -188,31 +193,45 @@ const Create = () => {
                     id={id}
                     key={id}
                     title={
+                      // @ts-expect-error TODO: Fix type error
                       event?.name?.[i18n.language] ??
+                      // @ts-expect-error TODO: Fix type error
                       event?.name?.[event.mainLanguage]
                     }
                     locationName={
+                      // @ts-expect-error TODO: Fix type error
                       event?.location?.name?.[i18n.language] ??
+                      // @ts-expect-error TODO: Fix type error
                       event?.location?.name?.[event.location.mainLanguage]
                     }
                     locationCity={
+                      // @ts-expect-error TODO: Fix type error
                       event?.location?.address?.[i18n.language]
                         ?.addressLocality ??
+                      // @ts-expect-error TODO: Fix type error
                       event?.location?.address?.[event.location.mainLanguage]
                         ?.addressLocality
                     }
                     organizerName={
+                      // @ts-expect-error TODO: Fix type error
                       event?.organizer?.name?.[i18n.language] ??
+                      // @ts-expect-error TODO: Fix type error
                       event?.organizer?.name?.[event.location.mainLanguage]
                     }
+                    // @ts-expect-error TODO: Fix type error
                     terms={event?.terms}
                     flex={1}
+                    // @ts-expect-error TODO: Fix type error
                     imageUrl={event?.image}
                     description={
+                      // @ts-expect-error TODO: Fix type error
                       event?.description?.[i18n.language] ??
+                      // @ts-expect-error TODO: Fix type error
                       event?.description?.[event.mainLanguage]
                     }
+                    // @ts-expect-error TODO: Fix type error
                     productionName={event?.production?.title}
+                    // @ts-expect-error TODO: Fix type error
```

Comment:
This is maybe counterproductive as we're likely not going to fix these one by one and it hinders readability in the meantime, is there an equivalent to the file-level nocheck for expect-error maybe? :s